Dockerfiles/midstream/unit_tests.py

```diff
@@ -80,8 +80,9 @@ def test_negative_image_to_test_not_set(self):
                       "Result code not found in %s" % message)
         self.assertEqual(102, result.returncode)
         self.assertTrue(os.path.exists(OUTPUT_DIR+".errormessage"))
-    
-    # Run with a test-operator which passes the bundle image validation job
+
+    # Run with a test-operator which fails the deprecated image check in the
+    # bundle image validation job
     def test_default_positive(self):
```

Comment:
Can you change the name of this test and adjust the assertions on line 92? Currently this unit test is not executed because it has the same name as the one below it.